fw/tls.c

```diff
@@ -94,6 +94,11 @@ tfw_tls_connection_recv(TfwConn *conn, struct sk_buff *skb)
 	default:
 	case T_BAD:
 		r = T_BAD;
+		switch (tls->conf->endpoint) {
+		case TTLS_IS_SERVER:
+			TFW_INC_STAT_BH(serv.tls_hs_failed);
+			break;
+		}
```

Comment:
Maybe we can use `if` here, instead of switch with single case?

Reply:
Fixed